expression/aggregation/agg_to_pb_test.go

```diff
@@ -65,7 +66,7 @@ func TestAggFunc2Pb(t *testing.T) {
 			aggFunc, err := NewAggFuncDesc(ctx, funcName, args, hasDistinct)
 			require.NoError(t, err)
 			aggFunc.RetTp = funcTypes[i]
-			pbExpr := AggFuncToPBExpr(ctx, client, aggFunc)
+			pbExpr, _ := AggFuncToPBExpr(ctx, client, aggFunc, kv.UnSpecified)
```

Comment:
add `require.NoError` to make sure that `AggFuncToPBExpr` does not return error?

Reply:
ok